local-testing-support/keycloak-docker/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
+services:
+  keycloak:
+    image: keycloak-1.9.1:latest
+    ports:
+      - '8080:8080'
+      - '9990:9990'
+    volumes:
+      # Set this to where you unzipped KeyCloak.
+      - /Users/giontc1/Documents/MICrONS/keycloak-1.9.1.Final:/srv/keycloak
```

Comment:
hard coded with your home directory.

Reply:
Yes, comment directs user to replace with the location of KeyCloak on their machine.